squireChuck/interview-javascript-check
# Ijc3 Add typescript support

Ticket #3


## Testing

automated - run `npm test` and ensure all tests pass.

manual:
- Run `npm start` - verify that `Hi World` is printed.
- Run `npm start -- --help` - verify script usage and examples are printed.
- Run `npm start -- --greeting Greetings` - verify `Greetings World` is printed.
- Run `npm start -- --name Jack` - verify `Hi Jack` is printed.
- Run `npm start -- --greeting Greetings --name Jack` - verify `Greetings Jack` is printed.
